src/utils/SortUtil.ts

```diff
@@ -17,5 +17,11 @@ export const sortBy =
       return -1; // Place undefined at the end
     }
 
-    return fieldA > fieldB ? order : -order;
+    // Check if both values are strings
+    if (typeof fieldA === "string" && typeof fieldB === "string") {
+      return fieldA.localeCompare(fieldB) * order;
```

Comment:
By default **localCompare** does not guarantee case insensitive ordering.
According to [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator#sensitivity) use **base** option or if you prefear you can compare the two strings after toLowerCase()/toUpperCase() convertion

Reply:
We should not use toLowerCase/toUpperCase for the following reason: localeCompare is the recommended approach when sorting JS arrays as it ensures that your sorting respects the nuances of different languages and character sets 
(takes into account the locale).

Added base fix tho.

Reply:
I suggested using loweCase/upperCase with the localCompare function as-is.
`fieldA.toLowerCase().localeCompare(fieldB.toLowerCase())`.
BTW with parameter as you did is the best choice.

Reply:
this is probably the absolute best
`fieldA.toLowerCase().localeCompare(fieldB.toLowerCase())`
but at this point i think we can do fine with the current code, right?

Reply:
For me is equivalent, so use the version you prefear

Reply:
if you are convinced, can you approve the PR?